.github/workflows/winget.yml

```diff
@@ -0,0 +1,12 @@
+name: Publish to WinGet
+on:
+  release:
+    types: [released]
+jobs:
+  publish:
+    runs-on: windows-latest # action can only be run on windows
+    steps:
+      - uses: vedantmgoyal2009/winget-releaser@latest
+        with:
+          identifier: OliverSchwendener.ueli
+          token: ${{ secrets.GH_TOKEN }}
```

Comment:
Can you explain why this environment variable is needed?

Reply:
To create pull requests at https://github.com/microsoft/winget-pkgs.